testing/cert.go

```diff
@@ -30,6 +34,9 @@ var (
 	// Other valid test certs different from the default.
 	OtherCACert, OtherCAKey        = chooseGeneratedOtherCA()
 	OtherCACertX509, OtherCAKeyRSA = mustParseCertAndKey(OtherCACert, OtherCAKey)
+
+	// SSHServerHostKey for testing
+	SSHServerHostKey = mustGenerateSSHServerHostKey()
```

Comment:
If this is going to be in the testing package, pre-generate one. Or, better yet, change this to a function that uses sync.Once to only generate the test key once on first use.

Reply:
Done!